Sources/GDBRemote/Session.cpp

```diff
@@ -1957,6 +1958,31 @@ void Session::Handle_qP(ProtocolInterpreter::Handler const &,
 }
 
 //
+// Packet:        qPlatform_chmod:mode,path
+// Description:   Change permissions of a file on the remote.
+// Compatibility: LLDB
+//
+void Session::Handle_qPlatform_chmod(ProtocolInterpreter::Handler const &,
+                                     std::string const &args) {
+  char *eptr;
+  uint32_t mode = std::strtoul(args.c_str(), &eptr, 16);
+  if (*eptr++ != ',') {
+    sendError(kErrorInvalidArgument);
+    return;
+  }
+
```

Comment:
Are we sure than the string will be long enough ?

if (!*eptr) {
   // error
}